diff --git a/node_modules/expo-location/ios/LocationModule.swift b/node_modules/expo-location/ios/LocationModule.swift
index fe49dd8..baba93f 100644
--- a/node_modules/expo-location/ios/LocationModule.swift
+++ b/node_modules/expo-location/ios/LocationModule.swift
@@ -201,10 +201,6 @@ public final class LocationModule: Module {
       guard CLLocationManager.isMonitoringAvailable(for: CLCircularRegion.self) else {
         throw Exceptions.GeofencingUnavailable()
       }
-      guard try taskManager.hasBackgroundModeEnabled("location") else {
-        throw Exceptions.LocationUpdatesUnavailable()
-      }
-
       try taskManager.registerTask(withName: taskName, consumer: EXGeofencingTaskConsumer.self, options: options)
     }
 
diff --git a/node_modules/expo-location/ios/TaskConsumers/EXGeofencingTaskConsumer.m b/node_modules/expo-location/ios/TaskConsumers/EXGeofencingTaskConsumer.m
index 7e6482e..d5bd24c 100644
--- a/node_modules/expo-location/ios/TaskConsumers/EXGeofencingTaskConsumer.m
+++ b/node_modules/expo-location/ios/TaskConsumers/EXGeofencingTaskConsumer.m
@@ -71,7 +71,7 @@ - (void)startMonitoringRegionsForTask:(id<EXTaskInterface>)task
     self->_regionStates = regionStates;
 
     locationManager.delegate = self;
-    locationManager.allowsBackgroundLocationUpdates = YES;
+    locationManager.allowsBackgroundLocationUpdates = NO;
     locationManager.pausesLocationUpdatesAutomatically = NO;
 
     for (NSDictionary *regionDict in regions) {
